### PR TITLE
Android Forms activity needs to forward events to ActivityLifetimeListener

### PR DESCRIPTION
**Pull request by LeRondPoint**

### CODE REVIEW

File: MvvmCross-Forms/MvvmCross.Forms.iOS/Views/Renderers/MvxImageViewRenderer.cs [hunk 1]
```diff
@@ -0,0 +1,112 @@
+﻿using System.ComponentModel;
+using Mvvmcross.Forms.iOS.Views.Renderers;
+using Mvvmcross.Forms.Views;
+using MvvmCross.Platform.Platform;
+using UIKit;
+using Xamarin.Forms;
+using Xamarin.Forms.Internals;
+using Xamarin.Forms.Platform.iOS;
+using IosMvxImageView = MvvmCross.Binding.iOS.Views.MvxImageView;
```
Comment (martijn00) [inline MvvmCross-Forms/MvvmCross.Forms.iOS/Views/Renderers/MvxImageViewRenderer.cs]: Always start naming conventions with Mvx. So MvxIosImageView

File: MvvmCross-Forms/MvvmCross.Forms.iOS/Views/Renderers/MvxImageViewRenderer.cs [hunk 2]
```diff
@@ -0,0 +1,112 @@
...
+
+[assembly: ExportRenderer(typeof(MvxImageView), typeof(MvxImageViewRenderer))]
+namespace Mvvmcross.Forms.iOS.Views.Renderers
+{
+    class MvxImageViewRenderer : ImageRenderer
+    {
+        private IosMvxImageView _NativeControl { get; set; }
```
Comment (martijn00) [inline MvvmCross-Forms/MvvmCross.Forms.iOS/Views/Renderers/MvxImageViewRenderer.cs]: Private accessors should not start with capital letter. `_nativeControl`

File: MvvmCross/Binding/Droid/Views/MvxImageView.cs [hunk 3]
```diff
@@ -89,18 +90,18 @@ protected IMvxImageHelper<Bitmap> ImageHelper
             }
         }
 
-        public MvxImageView(Context context, IAttributeSet attrs, int defStyleAttr)
+        public MvxImageView(Context context, IAttributeSet attrs, int defStyleAttr, Action afterImageChangeAction = null)
```
Comment (martijn00) [inline MvvmCross/Binding/Droid/Views/MvxImageView.cs]: Can we rename this to `imageChanged` or something?

File: MvvmCross-Forms/MvvmCross.Forms.Droid/MyApplicationActivity.cs.txt [hunk 4]
```diff
@@ -13,8 +13,8 @@ using Xamarin.Forms.Platform.Android;
 namespace Namespace
 {
     [Activity(Label = "View for anyViewModel", ScreenOrientation=ScreenOrientation.Portrait)]
-    public class MvxFormsApplicationActivity
-        : FormsApplicationActivity
+    public class MyApplicationActivity
```
Comment (martijn00) [inline MvvmCross-Forms/MvvmCross.Forms.Droid/MyApplicationActivity.cs.txt]: Do we still need to keep this file?

File: MvvmCross/Binding/Droid/Views/MvxImageView.cs (hunk 3 above)
Comment (LeRondPoint) [inline MvvmCross/Binding/Droid/Views/MvxImageView.cs]: Of course, I used the variable name from the iOS version so I'll change both

**Comment by LeRondPoint** [comment]
Done

File: MvvmCross-Forms/MvvmCross.Forms.Droid/MvvmCross.Forms.Droid.csproj [hunk 5]
```diff
@@ -138,7 +149,7 @@
     </ProjectReference>
   </ItemGroup>
   <ItemGroup>
-    <None Include="MvxFormsApplicationActivity.cs.txt" />
+    <None Include="MyApplicationActivity.cs.txt" />
```
Comment (martijn00) [inline MvvmCross-Forms/MvvmCross.Forms.Droid/MvvmCross.Forms.Droid.csproj]: Also remove it from the csproj

**Comment by LeRondPoint** [comment]
Hay forgot to save!

**Comment by martijn00** [comment]
Don't worry, i fixed it.

**Pull request merged**